catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,7 +28,7 @@
         <property name="indexLocation" value="data/geonames-index"/>
     </bean>
 
-    <service ref="geonamesQueryable" interface="org.codice.ddf.spatial.geocoding.GeoEntryQueryable"/>
+    <service ref="geonamesQueryable" interface="org.codice.ddf.spatial.geocoding.GeoEntryQueryable" ranking="50"/>
```

Comment:
This is so that if there's a local lucene index set up for geonames, but the web service version also happens to be installed, it will prefer the local one.